src/export/ExportFFmpegDialogs.cpp

```diff
@@ -1878,6 +1887,7 @@ void ExportFFmpegOptions::PopulateOrExchange(ShuttleGui & S)
                S.EndMultiColumn();
                S.StartMultiColumn(4, wxALIGN_LEFT);
                {
+                  // PRL: This preference now does nothing!
```

Comment:
Sorry but... you should remove my comment now!  You fixed the use of tags.

Reply:
Missed it, sorry!